### PR TITLE
Amplitude session

### PR DESCRIPTION
**Pull request by migs647**
Added amplitude middleware for session tracking.

### CODE REVIEW

File: Pod/Classes/SEGAmplitudeSession.m [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+       }
+
+        if (newPayload != nil) {
+            ctx.payload = newPayload;
+        }
+
+    }];
+
+    next(changedContext);
+//    next(context);
```
Comment (migs647) [inline Pod/Classes/SEGAmplitudeSession.m:96]: Remove this line

**Review by briemcnally — APPROVED** [review]
LGTM

**Comment by calube** [comment]
Doesn't this integration already provide amplitude sessions? What does this PR add in terms of Amplitude sessions?

**Comment by migs647** [comment]
Closing this as this is not ready for production.

**Pull request closed without merging**